docs/env/atari.rst

```diff
@@ -25,7 +25,7 @@ Options
   ``env.step``, default to ``batch_size``;
 * ``seed (int)``: the environment seed, default to ``42``;
 * ``max_episode_steps (int)``: the maximum number of steps for one episode,
-  default to ``108000``;
+  default to ``27000``;
```

Comment:
default to ``27000``, which corresponds to 108000 frames or roughly 30 minutes of gameplay [(Hessel et al. 2018, Table 3)](https://arxiv.org/pdf/1710.02298.pdf) because of the 4 skipped frames ;